### PR TITLE
docs: add Git links

### PR DESCRIPTION
**Pull request by Lanson2232**
## What type of PR is this? (check all applicable)

- [x] Added new link to category
- [ ] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [ ] Documentation update
- [ ] Frontend update

## Description
added new links to Git category

### CODE REVIEW

**Comment by fatihlattab** [comment]
@YusufBilgin careful, it's main.

File: docs/development-tools/git.md [hunk 1]
```diff
@@ -10,7 +10,10 @@ sidebar_label: Git
 - [Türkçe Git 101 | Ali Öztürk](https://aliozgur.gitbooks.io/git101/content/ "Türkçe Git 101 | Ali Öztürk")
 - [🎥 Git Dersleri | Barış Aslan 🎥](https://youtu.be/rWG70T7fePg "Git Dersleri")
 - [🎥 Git & GitHub Crash Course For Beginners | Traversy Media 🎥](https://youtu.be/SWYqp7iY_Tc "Git & GitHub Crash Course For Beginners | Traversy Media")
-
+- [Yeni Başlayanlar için Git Dersleri | Tarık Güney (Youtube Playlist)](https://www.youtube.com/playlist?list=PL_Z0TaFYSF3IqQKPOmbigAOVMMlZ2yU4K "Yeni Başlayanlar için Git Dersleri")
+- [Learn Git in Y minutes](https://learnxinyminutes.com/docs/tr-tr/git-tr/ "Learn Git in Y minutes")
+- [Git Tutorial for Beginners | Programming with Mosh](https://www.youtube.com/watch?v=8JJ101D3knE&ab_channel=ProgrammingwithMosh "Git Tutorial for Beginners: Learn Git in 1 Hour")
+- [Learn the Basics of Git | freeCodeCamp](https://www.freecodecamp.org/news/learn-the-basics-of-git-in-under-10-minutes-da548267cc91/ "Learn the Basics of Git in Under 10 Minutes")
```
Comment (YusufBilgin) [inline docs/development-tools/git.md:16]:
```suggestion
- [Learn the Basics of Git | freeCodeCamp](https://www.freecodecamp.org/news/learn-the-basics-of-git-in-under-10-minutes-da548267cc91/ "Learn the Basics of Git in Under 10 Minutes")

```

**Comment by YusufBilgin** [comment]
👍

**Review by YusufBilgin — APPROVED** [review]
👍

**Pull request merged**